<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mysqlManager</class>
 <widget class="QMainWindow" name="mysqlManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>741</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="status">
     <attribute name="title">
      <string>电脑状态</string>
     </attribute>
     <widget class="QTableWidget" name="tableStatus">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>391</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>型号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>电脑状态</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="buttonAssign">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>分配</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRepair">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>报修</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeCom">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>420</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>移除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="comPushUp">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>上移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="comPushDown">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>下移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonQuery">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>320</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>编号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qNumber"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>360</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>型号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qModel"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>400</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>状态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="qStatus">
         <item>
          <property name="text">
           <string>不指定</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>关机</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>使用中</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="buttonReset">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonShut">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>关机</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="user">
     <attribute name="title">
      <string>用户</string>
     </attribute>
     <widget class="QPushButton" name="buttonRecharge">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>260</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableVIP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>411</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>身份证号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>卡号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>余额</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>会员等级</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>用户身份证号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="id2Card"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="qCard">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="useRecord">
     <attribute name="title">
      <string>上机记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableRecord">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>631</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>流水号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>用户名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>卡号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>电脑编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束时间</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="queryRecord">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>流水号  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qRecord"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>用户名  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qUser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>卡号    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qVIP"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>电脑编号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="qCom"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="repairRecord">
     <attribute name="title">
      <string>报修记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableRepair">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>591</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>报修流水号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>报修电脑编号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>报修时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>报修原因</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>修理状态</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="qRepair">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mRepair">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>更改</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>流水号  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="repairID"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>编号    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="repairNum"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>修理状态</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="repairSta"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>报修原因</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="repairReason">
         <item>
          <property name="text">
           <string>任意</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>硬件故障</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>软件故障</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="addComputer"/>
    <addaction name="addCustomer"/>
    <addaction name="addCard"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addComputer">
   <property name="text">
    <string>New Computer</string>
   </property>
  </action>
  <action name="addCustomer">
   <property name="text">
    <string>New Customer</string>
   </property>
  </action>
  <action name="addCard">
   <property name="text">
    <string>New Card</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
